shells/android/javatests/arcs/storage/service/StorageServiceConnectionTest.kt

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.storage.service
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import arcs.crdt.CrdtCount
+import arcs.crdt.parcelables.ParcelableCrdtType
+import arcs.data.CountType
+import arcs.storage.ExistenceCriteria
+import arcs.storage.StoreOptions
+import arcs.storage.driver.RamDiskStorageKey
+import arcs.storage.parcelables.ParcelableProxyMessage
+import arcs.storage.parcelables.toParcelable
+import arcs.testutil.assertSuspendingThrows
+import com.google.common.truth.Truth.assertThat
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.eq
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/** Tests for [StorageServiceConnection]. */
+@RunWith(AndroidJUnit4::class)
+@UseExperimental(ExperimentalCoroutinesApi::class)
+class StorageServiceConnectionTest {
+    private lateinit var delegateMock: StorageServiceBindingDelegate
+    private lateinit var serviceMock: IStorageService
+
+    @Before
+    fun setup() {
+        delegateMock = mock {
+            on { bindStorageService(any(), any(), any()) }.doReturn(true)
```

Comment:
Do you mind running a Copybara presubmit on this PR? I had issues with another test file that used mockito -- I had to disable it. I'm curious if it's just the specifics of that test and how it's using mockito, or if it's mockito in general.